options.go

```diff
@@ -70,6 +70,13 @@ type RequestOptions struct {
 		JSON     bool        `long:"json" hidden:"true"`
 		Raw      bool        `long:"raw" hidden:"true"`
 	}
+
+	StreamRequestOptions StreamRequestOptions
+}
+
+// StreamRequestOptions are stream request related options
+type StreamRequestOptions struct {
+	Interval timeMillisFlag `long:"stream-interval" description:"Interval between consecutive stream request message sends."`
```

Comment:
Can you elaborate this comment and explain how it is working with concurrency and number of connections please

Reply:
updated the description
```
Interval between consecutive stream message sends, applicable separately to every stream request opened on a connection.
```